packages/cli/src/commands/generate/page/templates/page.tsx.template

```diff
@@ -7,6 +8,10 @@ type ${pascalName}PageProps = {
 const ${pascalName}Page = (<%- paramName === '' ? '' : `${propParam}: ${pascalName}PageProps` %>) => {
   return (
     <>
+      <MetaTags
+        title="${pascalName}"
+        description="${pascalName} description"
```

Comment:
For SEO, it's _worse_ to have a bad description (like this) than to have no description at all — indexers will use the page content if no description.
1. comment this out by default
2. add a line or two about "you should un-comment and add a unique description, 155 characters or less". Possible help content to link to for best practices (pick one? I like yoast the best):
    - https://yoast.com/meta-descriptions/
    - https://moz.com/learn/seo/meta-description
    - https://developers.google.com/search/docs/advanced/appearance/good-titles-snippets

Reply:
I think we should with a doc with Creative Commons license like the one of Google. Also, this one have multi languages and updated often